main.go

```diff
@@ -40,28 +40,53 @@ func main() {
 	flag.Parse()
 	cmd := flag.Arg(0)
 
+	// Get any potential trailing args
+	args := flag.Args()
+	if len(args) > 1 {
+		args = args[1:]
+	} else {
+		args = nil
+	}
+
 	switch cmd {
 	case "update":
-		out.Notification("Updating packages")
-		if err = v.updatePlugins(); err != nil {
+		if len(args) > 0 {
+			out.Notificationf("Updating %v", args)
+		} else {
+			out.Notification("Updating packages")
+		}
+		if err = v.updatePlugins(args...); err != nil {
 			handleError(err)
 		}
 
 		out.Success("Update complete")
 
 	case "build":
-		out.Notification("Building packages")
-		if err = v.buildPlugins(); err != nil {
+		if len(args) > 0 {
+			out.Notificationf("Building %v", args)
+		} else {
+			out.Notification("Building packages")
+		}
+		if err = v.buildPlugins(args...); err != nil {
```

Comment:
Add newline please

Reply:
Maybe we should make a helper func out of this pattern. I've seen it twice so far